### PR TITLE
[build]: add option add timestamp in build log

### PR DESCRIPTION
**Pull request by lguohan**
…ld log

Signed-off-by: Guohan Lu <lguohan@gmail.com>

**- What I did**
add SONIC_CONFIG_BUILD_LOG_TIMESTAMP to add timestamp in build log

**- How I did it**

**- How to verify it**
```
add timestamp in each job build log
    
    example:
    
       [01:39:21] dh clean  --with autotools-dev
       [01:39:22]    dh_auto_clean
       [01:39:27]      make -j16 distclean
```
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest vs please

File: scripts/process_log.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+addts=""
```
Comment (jleveque) [inline scripts/process_log.sh]: Suggest making variable name more understandable. E.g., `add_timestamp`

**Comment by lguohan** [comment]
retest mellanox

**Comment by lguohan** [comment]
retest mellanox please

**Comment by lguohan** [comment]
retest broadcom please

**Comment by lguohan** [comment]
retest vsimage please

**Comment by lguohan** [comment]
retest broadcom please

**Comment by lguohan** [comment]
retest mellanox please

**Pull request merged**